pyton_hw.py

```diff
@@ -0,0 +1,47 @@
+import random
+
+
+def play_coin_flip():
+    player_wins = 0
+    player_losses = 0
+
+    while True:
+        player_choice = input("Виберіть Орел (O) або Решка (Р), або натисніть (X) щоб завершити гру: ")
+        player_choice = player_choice.upper()
+
+        if player_choice == 'X':
+            break
+
+        while player_choice not in ['O', 'P']:
+            player_choice = input("Виберіть Орел (O) або Решка (Р), або натисніть (X) щоб завершити гру: ")
+            player_choice = player_choice.upper()
+
+        if player_choice == 'O':
+            player_choice_name = 'Орел'
+        else:
+            player_choice_name = 'Решка'
+
+        options = ['O', 'P']
+        computer_choice = random.choice(options)
+
+        if computer_choice == 'O':
+            computer_choice_name = 'Орел'
+        else:
+            computer_choice_name = 'Решка'
+
+        print("Ваш вибір:", player_choice_name)
+        print("Вибір комп'ютера:", computer_choice_name)
```

Comment:
Computer choise? 
This is overthinking of game :) 
It's just player chooses the Heads or Tails. You pick randomly one of two. If random choice == players, then it's a win. Else = lose.
Computer should make a chose. 
Maybe just remove this line or re-phares/re-name the variables.